### PR TITLE
Meeting summary for 2016-04-07

### PR DESCRIPTION
**Pull request by G1lius**

### CODE REVIEW

**Comment by MarcoFalke** [comment]
ack

File: _posts/en/meetings/2016-04-07-meeting.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+version: 1
+---
+{% include _toc.html %}
+
+- [Link to this week logs](http://bitcoinstats.com/irc/bitcoin-core-dev/logs/2016/04/07#l1460055658.0)
+- [Meeting minutes by meetbot](http://www.erisian.com.au/meetbot/bitcoin-core-dev/2016/bitcoin-core-dev.2016-04-07-19.00.html)
+
+---
+
+## Main topics
```
Comment (btcdrak) [inline _posts/en/meetings/2016-04-07-meeting.md]: Needs blank line after title to render properly.

Comment (G1lius) [inline _posts/en/meetings/2016-04-07-meeting.md]: Fixed.

**Comment by btcdrak** [comment]
merged.

**Pull request closed without merging**